packages/flutter_tools/test/general.shard/android/android_studio_test.dart

```diff
@@ -686,6 +809,48 @@ void main() {
       Platform: () => linuxPlatform,
       ProcessManager: () => FakeProcessManager.any(),
     });
+
+    testUsingContext('Can find Android Studio 2020.3 bundled Java version on Linux', () {
+      const String studioHomeFilePath =
+          '$homeLinux/.cache/Google/AndroidStudio2020.3/.home';
```

Comment:
```suggestion
      const String studioHomeFilePath = '$homeLinux/.cache/Google/AndroidStudio2020.3/.home';
```

Reply:
Thanks for your suggestion, but this suggested change doesn't seem to do anything?

Reply:
This is using two lines, it can be a single line.